File: Python/calckill.py
# Chris Sikes
# Kill calculator during hours 9pm and 10pm every 5 seconds, else don't
# 1/28/21

import os
import time
from datetime import datetime

while True:
    hour = (datetime.now().strftime("%H"))
    if((hour == "21") or (hour == "22")):
        os.system("taskkill /im win32calc.exe")
    time.sleep(5)

File: C/calckill.c
#include <unistd.h> 
#include <time.h>

int main(void)
{
int hour;

while (1){
time_t t = time(NULL);
struct tm tm = *localtime(&t);
hour = tm.tm_hour;
if (hour == 21 || hour == 22) {
system("taskkill /F /IM win32calc.exe");
}
sleep(5);
      

}
return 0;
   
}

